android/src/main/java/com/geektime/rnonesignalandroid/RNOneSignal.java

```diff
@@ -538,6 +538,7 @@ public void sendOutcome(final String name, final Callback callback) {
          public void onSuccess(OSOutcomeEvent outcomeEvent) {
             if (outcomeEvent == null)
                callback.invoke(new WritableNativeMap());
```

Comment:
The null behavior differs from what is done in iOS.
https://github.com/OneSignal/react-native-onesignal/pull/1129/files#diff-153b24fed9bd79fc91c7ca30ccccdd37f29859785574f541911460e7a4cbdd6cR392

There we do NOT fire the callback at all if it is `null`. The behavior should match between Android and iOS.